spec/system/reports_spec.rb

```diff
@@ -4,7 +4,7 @@
   let!(:user) { create(:user, name: 'ケン') }
 
   before do
-    login_as(user)
+    login_as(user, user.password)
```

Comment:
```suggestion
    login_as(user, "passabcd1234")
```

または

```suggestion
    login_as(user)
```

僕だったら後者にするかな。

基本的にpasswordは再取得できないものとして一貫したコードを書く方がベターです。
（コンテキストによってpasswordを取得できる・できないを切り替えないといけないのは脳への負担が大きい）